Source/WebCore/dom/Element.cpp

```diff
@@ -2162,7 +2132,7 @@ ExceptionOr<void> Element::setAttribute(const AtomString& qualifiedName, const T
         AttributeTypeAndSink attributeTypeAndSink;
         if (document().requiresTrustedTypes())
             attributeTypeAndSink = trustedTypeForAttribute(nodeName(), name.localName().convertToASCIILowercase(), this->namespaceURI(), name.namespaceURI());
-        auto attributeValue = trustedTypesCompliantAttributeValue(attributeTypeAndSink.attributeType, value, this, attributeTypeAndSink.sink);
+        auto attributeValue = trustedTypesCompliantAttributeValue(*document().scriptExecutionContext(), attributeTypeAndSink.attributeType, value, attributeTypeAndSink.sink);
```

Comment:
Similar changes here could make this code clearer and simpler too.
```suggestion
        auto compliantValue = trustedTypesCompliantAttributeValue(document(), type.attributeType, value, type.sink);
```